app/models/user.rb

```diff
@@ -27,14 +30,22 @@ class User < Common::RedisStore
   attribute :participant_id
   attribute :ssn
 
+  # mvi 'golden record' data
+  attribute :mvi
+
   validates :uuid, presence: true
   validates :email, presence: true
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :birth_date, presence: true
+  validates :ssn, presence: true, format: /\d{3}-\d{2}-\d{4}/
```

Comment:
For EVSS, I need to have the SSN as a series of 9 characters. Can we save it as a plain string and then convert it to the form with the dashes as needed? Seems simpler to do that way. Either way should be fine, I guess, but the main thing is that we need to document which way it is so that we can be consistent with it.